source/platforms/brev-dev.md

```diff
@@ -0,0 +1,82 @@
+## Section 1: Setting up your instance
+
+Navigate to the Brev.dev console and click on "Create your first instance"
+
+![Alt Text](/_static/images/platforms/brev/brev1.png)
+
+Select Container Mode
```

Comment:
When calling out specific buttons or tabs can you put them in quotes to help folks know they are looking for a specific text.

```suggestion
Select "Container Mode"
```

Reply:
This still needs quotes